atlascope/tests/test_core_endpoints.py

```diff
@@ -140,6 +140,7 @@ def test_list_datasets(user_api_client, user, dataset_factory):
             'description': d.description,
             'source_uri': d.source_uri,
             'public': d.public,
+            'importer': d.importer,
```

Comment:
Does [`dataset_factory`](https://github.com/atlascope/atlascope-api/blob/85a62c7174ca6d077d53a28bc8400ba6d19cf49a/atlascope/tests/factories.py#L17-L25) need to be updated to have a faked importer?

Reply:
yes, absolutely. Good catch.